src/sentry/static/sentry/app/components/onboardingWizard/sidebar.tsx

```diff
@@ -148,7 +148,11 @@ class OnboardingWizardSidebar extends React.Component<Props> {
     ];
 
     return (
-      <TaskSidebarPanel collapsed={collapsed} hidePanel={onClose}>
+      <TaskSidebarPanel
+        collapsed={collapsed}
+        hidePanel={onClose}
```

Comment:
I think we should rename onClose to something more meaningful here...

Reply:
Yeah I agree, going to leave this as is though for this PR